MOCC/src/configurations/NewPyTango/demo_script.py

```diff
@@ -0,0 +1,35 @@
+from config_manager import ConfigManager
+
+if __name__ == '__main__':
+
+    # Connecting to ConfigManager
+    cf = ConfigManager('192.168.0.3', '8080')
```

Comment:
What is this link exactly?
What is the difference between this link and the 127.0.0.1:5000/ ?

Is ConfigManager a server as well?